src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -1201,3 +1201,8 @@
         az monitor metrics list-definitions --resource /subscriptions/{subscriptionID}/resourceGroups/Space1999/providers/Microsoft.Network/networkSecurityGroups/ADDS-NSG
     crafted: true
 """
+
+helps['monitor clone'] = """
+type: command
+short-summary: Clone metrics alert rules from one resource to another resource.
+"""
```

Comment:
Maybe we can add some examples?

Reply:
Sure

Reply:
missing examples?

---
In reply to: [391417892](https://github.com/Azure/azure-cli/pull/12233#discussion_r391417892) [](ancestors = 391417892)

Reply:
added